public/css/style.css

```diff
@@ -0,0 +1,85 @@
+/*
+*
+* ==========================================
+* ==========================================
+*
+*/
+
+.warning {
+  color: red;
+}
+
+body {
+  min-height: 50vh;
+  background-color: #e2f7ed;
+  background-image: linear-gradient(to right, #f3f7f5 0%, #bdddd7 100%);
+  font-style: "Courier New";
+}
+
+.title {
+  text-align: center;
+  color: #235e81;
+}
+
+.container {
+  text-align: center;
+  list-style: none;
+}
+
+.form {
+  text-align: center;
+  list-style: none;
+}
+
+.main-navigation {
+  border-top: 4px solid #000;
+  border-bottom: 2px solid #000;
+  padding-bottom: 100px;
+}
+
+/** Social Button CSS **/
+
+.share-buttons {
+  text-align: center;
+}
```

Comment:
Those two words **Share Buttons** better to delete (it is enough just btns with social links) or make smaller.

Reply:
I changed the class name, thanks for the feedback